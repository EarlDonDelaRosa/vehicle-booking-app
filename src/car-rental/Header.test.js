import Headers from "./Header";
import { render, screen } from '@testing-library/react';


describe('This is login page', () => {
    beforeEach(() => {
      render(<Headers />);
    })
  
    test('h2 Car Rental Service', () => {
      const linkElement = screen.getByText(/Car Rental Service/);
      expect(linkElement).toBeInTheDocument();
    });
  
    test('To check if data has changed', () => {
      const linkElement = screen.getByText(/Car Rental Service/);
      expect(linkElement).toMatchSnapshot();
    });
  })