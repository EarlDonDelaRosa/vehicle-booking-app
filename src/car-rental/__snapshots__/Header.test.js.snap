// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This is login page To check if data has changed 1`] = `
<h2
  align="left"
  class="ms-3"
>
  Car Rental Service 23
</h2>
`;
